appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -960,6 +963,12 @@ private void initializeUi() {
     // the project
     tutorialPanel.setVisible(false);
 
+    //Create Help Panel
+    helpPanel = new TutorialPanel();
```

Comment:
Here you create a TutorialPanel, but you've also added a HelpPanel class. Either remove the HelpPanel class definition since it's not used, or switch to using HelpPanel here.